### PR TITLE
feat: contribute

### PR DESCRIPTION
**Pull request by jiyuujin**
Add contribute

### CODE REVIEW

File: CONTRIBUTING.ja.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# コントリビュート
+
+あなたのコントリビュートをお待ちしております！
```
Comment (kikuchy) [inline CONTRIBUTING.ja.md:3]: 🙌

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Contribution
+
+We look forward to your contribution!
+
+## How to contribute
+
+### How to find a task
+
+I'm using GitHub Issue for task management. Find the Issue you want to contribute here. Pull Requests without issues are fine. In that case, please provide details such as the reason, cause, and solution in the Pull Request.
```
Comment (kikuchy) [inline CONTRIBUTING.md:9]:
```suggestion
We are using GitHub Issue for task management. Please find the Issue you want to contribute here. You can also make Pull Requests without relation to any issues. In this case, please provide details such as the reason, cause, and solution in the Pull Request.
```

I just feel using "we" is natural.
And the sentence `Pull Requests without issues are fine.` losing the subject, using "you" is better I think.

**Comment by mightyfrog** [comment]
この辺を参考にするとよいかも。

https://source.android.com/setup/contribute

File: CONTRIBUTING.ja.md (hunk 1 above)
Comment (jiyuujin) [reply CONTRIBUTING.ja.md:3]: @kikuchy Thanks🙏🏻

**Comment by jiyuujin** [comment]
@mightyfrog 

ずっと放置しててもアレなので一旦マージさせていただきますね

**Pull request merged**